### ISSUE TITLE
Wrong hint for installing missing apache2 dependencies on Ubuntu 16.04

### ISSUE THREAD

**Issue by acosonic** [phusion/passenger#1884]
When running

`passenger-install-apache2-module`

it produces following output:

```
* To install Apache 2 development headers:
   Please install it with apt-get install apache2-threaded-dev

```

And when:
`apt-get install apache2-threaded-dev`
is run, it gives:
`E: Unable to locate package apache2-threaded-dev`

The proper way to fix requirements on Ubuntu 16.04 would be:

`apt-get install apache2-dev`

**Comment by khamroddin** [comment]
Hi
I am also having difficulty while installing apache + passenger modulde
apt-get install apache2-threaded-dev
throws error:
E: Package 'apache2-threaded-dev' has no installation candidate

**Comment by OnixGH** [comment]
@khamroddin this issue was already fixed in Passenger 5.1.0.

**Comment by mtolava** [comment]
With the Install of apache2-dev I resolve the dependence.
